### PR TITLE
Provides Auto Complete for file-path

### PR DESCRIPTION
**Pull request by dipak-pawar**
* Providing auto-complete for file-path.
* Removed --docker-file-name option.

Note: Now we can't give url as a file-path which user can give for k8s or openshift with previous implementation. cc @bartoszmajsak

### CODE REVIEW

**Comment by dipak-pawar** [comment]
[asciicast](https://asciinema.org/a/107595)

File: src/main/java/org/jboss/forge/arquillian/command/core/ContainerSetupWizard.java [hunk 1]
```diff
@@ -53,7 +53,7 @@
     private Event<ContainerInstallEvent> installEvent;
 
     @Inject
-    @WithAttributes(shortName = 'f', label = "Container Adapter Type", type = InputType.DROPDOWN, required = false)
+    @WithAttributes(shortName = 'f', label = "Container Adapter Type", type = InputType.DROPDOWN)
```
Comment (bartoszmajsak) [inline src/main/java/org/jboss/forge/arquillian/command/core/ContainerSetupWizard.java:56]: How is this related to the subject of this PR?

**Review by bartoszmajsak — COMMENTED** [review]
Overall LGTM, just one question. Good job!

File: src/main/java/org/jboss/forge/arquillian/command/core/ContainerSetupWizard.java (hunk 1 above)
Comment (dipak-pawar) [reply src/main/java/org/jboss/forge/arquillian/command/core/ContainerSetupWizard.java:56]: this is redundant as by default required is false. so I removed it to make it more cleaner. This has nothing to do with this PR.

Comment (bartoszmajsak) [reply src/main/java/org/jboss/forge/arquillian/command/core/ContainerSetupWizard.java:56]: Ok, thx :) Boyscoutting for the win.
